05_sql/farmersmarket_homework2.sqbpro

```diff
@@ -1,31 +1,31 @@

+


```

Comment:
Your Quereis for both HW are vey messed and difficult to understand the question you are attemting, and I dont know Why this ^M is displaying evrywhere in your query.